src/lib/infra/updateEnv.task.ts

```diff
@@ -1,15 +1,21 @@
 import type {Task} from '@feltcoop/gro';
+import {spawnProcess} from '@feltcoop/util/process.js';
 import {ENV_FILE_DEV, ENV_FILE_PROD, updateEnv} from '$lib/server/env';
 
 export const task: Task = {
 	summary: 'write git hash to the appropriate environment variables file',
 	run: async ({fs, log, dev}) => {
 		const envFile = dev ? ENV_FILE_DEV : ENV_FILE_PROD;
 		const branch = (await fs.readFile('.git/HEAD', 'utf8')).trim().substring(5);
-		const [gitHashContents, currentEnvContents] = await Promise.all([
-			fs.readFile('.git/' + branch, 'utf8'),
-			fs.readFile(envFile, 'utf8'),
-		]);
+
+		let gitHashContents = '';
+		const buildResult = spawnProcess('git', ['show-ref', '-s', `${branch}`], {stdio: 'pipe'});
```

Comment:
```suggestion
		const buildResult = spawnProcess('git', ['show-ref', '-s', branch], {stdio: 'pipe'});
```

I think this is right?